generator/transformer/marketmap_transforms.go

```
@@ -42,7 +43,11 @@ func OverrideMinProviderCount() TransformMarketMap {
 		}
 		logger.Info("overriding min provider count")
 		for name, market := range mm.Markets {
-			market.Ticker.MinProviderCount = cfg.MinProviderCountOverride
+			if filter.MinProvidersThreshold < len(market.ProviderConfigs) {
+				market.Ticker.MinProviderCount = uint64(filter.MinProvidersThreshold)
```

golangci-lint (1.64.8)

[high] G115: integer overflow conversion int -> uint64

(gosec)